deps/graph-parser/src/logseq/graph_parser/test/docs_graph_helper.cljs

```diff
@@ -132,7 +132,80 @@
                 set))
         "Has correct namespaces")))
 
-(defn docs-graph-assertions
+(defn- query-assertions
```

Comment:
This ns is only for code that is used by both graph-parser and the frontend. Since this code isn't being used by the graph-parser, this should be moved to the frontend test ns

Reply:
They are the previous version of the graph-parser CLI test codes.

I'm going to move them to somewhere in `src/test/frontend`